alcs-frontend/src/app/services/application/application-document/application-document.service.ts

```diff
@@ -0,0 +1,52 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { firstValueFrom } from 'rxjs';
+import { environment } from '../../../../environments/environment';
+import { ToastService } from '../../toast/toast.service';
+import { ApplicationDocumentDto } from './application-document.dto';
+
+const MAX_FILE_SIZE = 104857600;
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ApplicationDocumentService {
+  private url = `${environment.apiRoot}/application-document`;
+
+  constructor(private http: HttpClient, private toastService: ToastService) {}
+
+  async list(fileNumber: string) {
+    return firstValueFrom(this.http.get<ApplicationDocumentDto[]>(`${this.url}/application/${fileNumber}`));
+  }
+
+  async upload(fileNumber: string, file: File) {
+    if (file.size > MAX_FILE_SIZE) {
+      this.toastService.showWarningToast('Maximum file size is 100MB, please choose a smaller file');
```

Comment:
the size can be calculated based on the MAX_FILE_SIZE variable, so this stays in sync with limits

Reply:
Done!